newIDE/app/src/UI/Search/UseSearchStructuredItem.js

```diff
@@ -144,14 +145,14 @@ export const useSearchStructuredItem = <
     chosenFilters,
     excludedTiers,
     defaultFirstSearchItemIds,
+    withoutShuffling,
```

Comment:
Is it possible to change it for a `shuffleResults` that is true by default. Those with-negation-named props are hard to read in control flows.